Model/Sales/TrackTraceHolder.php

```diff
@@ -180,12 +176,13 @@ public function convertDataFromMagentoToApi($magentoTrack, $options)
             ->setCity($address->getCity())
             ->setPhone($address->getTelephone())
             ->setEmail($address->getEmail())
-            ->setLabelDescription($shipment->getOrder()->getIncrementId())
+            ->setLabelDescription($magentoTrack->getShipment()->getOrder()->getIncrementId())
```

Comment:
`$magentoTrack->getShipment()` zou ikl in een $shipment stoppen

Reply:
Done